examples/typescript-interop/node/README.md

```diff
@@ -0,0 +1,64 @@
+# TypeScript Interop
+
+This is an example of calling Roc code from [TypeScript](https://www.typescriptlang.org/) on [Node.js](https://nodejs.org/en/).
+
+## Installation
+
+You'll need to have a C compiler installed, but most operating systems will have one already.
+(e.g. macOS has `clang` installed by default, Linux usually has GCC by default, etc.)
```

Comment:
nit: \`gcc\` to match \`clang\`

Reply:
Fair point, changed in 29caaf9d61b6f31a1201ee15f7ebcd870b1ec619